recipe/meta.yaml

```diff
@@ -17,25 +17,24 @@ build:
     script: python setup.py install --single-version-externally-managed --record=record.txt
     entry_points:
         - celery = celery.__main__:main
-        - celeryd = celery.__main__:_compat_worker
-        - celerybeat = celery.__main__:_compat_beat
-        - celeryd-multi = celery.__main__:_compat_multi
 
 requirements:
     build:
         - python
         - setuptools
-        - pytz
-        - billiard >=3.3.0.23,<3.4
-        - kombu >=3.0.34,<4.0
     run:
         - python
         - pytz
-        - billiard >=3.3.0.23,<3.4
-        - kombu >=3.0.34,<4.0
+        - billiard >=3.5.0.2,<3.6
+        - kombu >=4.0,<5.0
```

Comment:
I was going from the 4.0.0 version here [default.txt](https://github.com/celery/celery/blob/v4.0.0/requirements/default.txt). The goal was to get 4.0.1, and 4.0.2 after we were okay with the changes.